inception-project/inception
# #283 - Resize span

**What's in the PR**
- Add resizing functionality in brat editor
- Add undo functionality for resizing actions
- Fix drag not ending when mouse is released outside event control scope

**How to test manually**
* When hovering with the mouse over an annotation bubble in the brat editor, orange handles appear on the left and right side. If the bubble is split, they will appear on the first/last bubble respectively - try grabbing them with the mouse and dragging them to change the annotation size

**Automatic testing**

**Documentation**
